File: ai-leak-detection/generate_copies.py
names = ['Ramesh', 'Suresh', 'Mahesh']
centers = ['Center_Kanpur', 'Center_Lucknow', 'Center_Delhi']
ref_codes = ['KNP7', 'LKO7', 'DEL7']

with open('master_paper.txt', 'r') as f:
    master_text = f.read()

for i in range(3):
    copy_text = master_text.replace('Ramesh', names[i])
    copy_text = copy_text + f'\n\nRef: {ref_codes[i]}'
    filename = f'copy_{i+1}.txt'
    with open(filename, 'w') as f:
        f.write(copy_text)
    print(f'{filename} created for {centers[i]} with ref code {ref_codes[i]}')

File: ai-leak-detection/generateCopies.js
import fs from 'fs';

const names = ['Ramesh', 'Suresh', 'Mahesh'];
const centers = ['Center_Kanpur', 'Center_Lucknow', 'Center_Delhi'];
const refCodes = ['KNP7', 'LKO7', 'DEL7'];

const masterText = fs.readFileSync('master_paper.txt', 'utf-8');

for (let i = 0; i < 3; i++) {
  let copyText = masterText.replace('Ramesh', names[i]);
  copyText = copyText + `\n\nRef: ${refCodes[i]}`;
  const filename = `copy_${i + 1}.txt`;
  fs.writeFileSync(filename, copyText);
  console.log(`${filename} created for ${centers[i]} with ref code ${refCodes[i]}`);
}